### DISCUSSION TITLE
How do we show subtitles during the trimming/editing process?

### DISCUSSION THREAD

**Discussion by dfg555** [mifi/lossless-cut#1935] [Q&A] [answered]
I can enable/disable them upon exporting/remuxing but I want to be able to see them during the editing/trimming process too? Is it possible?

**Comment by mifi** [comment, marked as answer]
yes, click on the subtitle icon in the preview

<img width="32" alt="Screenshot 2024-03-24 at 17 08 20" src="https://github.com/mifi/lossless-cut/assets/402547/df856740-8def-4a68-9489-44c15feec16c">

**Reply by dfg555** [reply]
Thanks I see it now.
